data/json/professions.json

```diff
@@ -35,6 +35,19 @@
       { "item": "p320mag_17rd_9x19mm", "ammo-item": "9mmfmj", "charges": 17 }
     ]
   },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "six_pack_beer",
+    "entries": [
+      { "item": "beer" },
```

Comment:
I would just add this to the profession items like `{ "item": "beer", "count": 6 }` much cleaner

also make sure all those beers fit in the character's inventory, add a sling pack or something that fits the theme